_usage/limitations-of-babelfish.md

```diff
@@ -302,21 +304,77 @@ features are discovered, and additional features are added to MS SQL:
 | `WAITFOR/RECEIVE` | This syntax is not supported. |
 | `WAITFOR TIME` | This syntax is not supported. |
 | `WITH ENCRYPTION` clause | This syntax is not supported for functions, procedures, triggers, or views. |
-| `CREATE/ALTER/DROP WORKLOAD CLASSIFIER` | Functionality related to this syntax is not supported |
+| `CREATE/ALTER/DROP WORKLOAD CLASSIFIER` | Functionality related to this syntax is not supported. |
 | `CREATE/DROP WORKLOAD CLASSIFIER` | This syntax is not supported. |
 | `WRITETEXT` | This syntax is not supported. |
 | `OPENXML()` | This function is not supported. |
-| XML datatype with schema (xmlschema) | XML type without schema is supported |
-| XML methods | This includes .VALUES(), .NODES(), and other methods |
+| XML datatype with schema (xmlschema) | XML type without schema is supported. |
+| XML methods | This includes .VALUES(), .NODES(), and other methods. |
 | XML indexes | XML indexes are not supported. |
 | XPATH expressions | This syntax is not supported. |
 | WITH XMLNAMESPACES construct | This syntax is not supported. |
 | `WITHOUT SCHEMABINDING` clause | Not supported in functions, procedures, triggers, or views. The object will be created, but as if `WITH SCHEMABINDING` was specified. |
 | `CREATE/ALTER/DROP SELECTIVE XML INDEX` clause | This syntax is not supported. |
 | `CREATE/ALTER/DROP XML SCHEMA COLLECTION` | This syntax is not supported. |
 
-However, more is missing than just features. There are also some
-corner cases which will provide the end user with [different
-behavior](/docs/usage/missing-features) than otherwise expected from MS SQL. It is
-important to understand those corner cases, as well.
+However, more is different than just features.
+For example, backup and recovery work differently in SQL Server and PostgreSQL,
+and you will have to use the PostgreSQL tools to backup Babelfish.
+It is important to understand such differences as well.
 
+## Dealing with limitations
+
+In this section, we've written a lot about limitations, unsupported features and
+such. However, how do you deal with these things in real life? If Babelfish
+runs a SQL statement that doesn't return a result set that is identical to the
+result set when run against SQL Server, the SQL statement will fail.
+To better deal with statements that might fail, Babelfish allows you to define
+*escape hatches*. An escape hatch is a flag that specifies Babelfish behavior
+when it encounters an unsupported feature or syntax. Babelfish can either return
+an error or ignore the condition if the PostgreSQL result set is not a perfect
+match for the result set returned by SQL Server.
+
+You can use the `sp_babelfish_configure` stored procedure to control
+the settings of each escape hatch. Use the script to specify if each escape
+hatch should be set to `ignored` or `strict`.
+
+If set to `strict`, Babelfish will return an error that you must
+correct before continuing. Include the `cluster` keyword to apply
```

Comment:
Again, I'll take it on faith. Changed as you say.

Reply:
I had hoped/expected that these SQL statements showing up in the docs would be tested against the actual product. Otherwise, how can you know the right thing is being documented? When you try that, it will be clear that 'cluster' is the correct option.